templates/superset_config.py

```diff
@@ -101,12 +101,10 @@ class CeleryConfig(object):
 TIME_ROTATE_LOG_LEVEL = "DEBUG"
 ENABLE_TIME_ROTATE = True
 
-# html santization
-HTML_SANITIZATION = os.getenv("HTML_SANITIZATION")
-sanitization_extention = os.getenv("HTML_SANITIZATION_SCHEMA_EXTENSIONS")
-
-if HTML_SANITIZATION and sanitization_extention:
-    HTML_SANITIZATION_SCHEMA_EXTENSIONS = sanitization_extention
+# html sanitization
+HTML_SANITIZATION = os.getenv("HTML_SANITIZATION").lower() != "false"
+if HTML_SANITIZATION:
+    HTML_SANITIZATION_SCHEMA_EXTENSIONS = os.getenv("HTML_SANITIZATION_SCHEMA_EXTENSIONS", {})
```

Comment:
I don't like conditionally defined variables, is it possible to remove the if condition here so that the variable is always in scope?

Reply:
Yes, I believe with this default that should not be an issue.

Reply:
Updated.